install_dependencies.js

```diff
@@ -4,7 +4,7 @@ var isWin = !!process.platform.match(/^win/);
 console.log();
 
 if (isWin) {
-  var cmd = spawn('npm.cmd', ['install', 'edge'], { cwd: __dirname });
+  var cmd = spawn('cmd', ['/C', 'npm.cmd', 'install', 'edge'], { cwd: __dirname });
```

Comment:
Another way would be as mentioned https://github.com/nodejs/node/issues/52554#issuecomment-2060026269

```suggestion
  var cmd = spawn('npm.cmd', ['install', 'edge'], { cwd: __dirname, shell: true });
```